crates/matrix-sdk-base/src/store/integration_tests.rs

```diff
@@ -91,6 +95,8 @@ pub trait StateStoreIntegrationTests {
     async fn test_send_queue(&self);
     /// Test saving/restoring server capabilities.
     async fn test_server_capabilities_saving(&self);
+    /// Test saving live location share beacons
+    async fn test_beacon_saving(&self);
```

Comment:
Fwiw, I don't think this test is useful, since it's testing whether state events are properly saved in all our state store implementations. That's been tested elsewhere, and if it wasn't working we would know by now, so please let's remove the changes in this file.

Reply:
OK, in which case please ignore my comments in this file :-)